packages/nuka/cypress/integration/carousel.spec.js

```diff
@@ -89,12 +96,21 @@ context('Carousel', () => {
         .first()
         .should('have.attr', 'data-slide', 'Slide 5');
 
+      cy.get('button').contains('Prev').should('not.be.disabled');
+      cy.get('button').contains('Next').should('not.be.disabled').click();
+
+      cy.get('.slide.slide-visible')
+        .should('have.length', 1)
+        .find('img')
+        .should('have.attr', 'data-slide', 'Slide 6');
```

Comment:
This modification corresponds to the change in behavior in the scenario laid out in the PR description.